code/modules/mob/living/carbon/human/species/station.dm

```diff
@@ -527,9 +527,11 @@
 		"r_foot" = list("path" = /obj/item/organ/external/foot/right/ipc)
 		)
 
-	abilities = list(/mob/living/carbon/human/proc/change_monitor,)
```

Comment:
It can be fixed by a shitty snowflake addition of usr.stat in the if check. But as I said, it's snowflakey, and then BYOND is weirded out about the fact that species datum isn't defined for client.